apps/frontend/src/constants/links.ts

```diff
@@ -46,3 +46,12 @@ export const WIKI_LINKS = {
 };
 
 export const HELPDESK_LINK = 'https://help.sovryn.app/';
+
+export const MAX_STAKING_APR_LINK =
+  'https://redash.sovryn.app/public/dashboards/VqJnymylogSI6jMHZPJIyog9RUJjUUS81cxYfNUM';
+
+export const MAX_STAKING_APR_API_LINK =
+  'https://redash.sovryn.app/api/queries/310/results.json?api_key=SSlE3jXGWFC4Bk3MeWcRIJiduSdeJ8ThQoaqltk9';
```

Comment:
Is this key is restricted enough to exposed like this @soulBit ?

Reply:
we will move it to the env file